mlir/lib/Dialect/Linalg/TransformOps/LinalgTransformOps.cpp

```diff
@@ -630,13 +710,25 @@ DiagnosedSilenceableFailure
 transform::FuseOp::apply(transform::TransformRewriter &rewriter,
                          mlir::transform::TransformResults &transformResults,
                          mlir::transform::TransformState &state) {
-  SmallVector<int64_t> tileSizes =
-      extractFromIntegerArrayAttr<int64_t>(getTileSizes());
-  SmallVector<int64_t> tileInterchange =
-      extractFromIntegerArrayAttr<int64_t>(getTileInterchange());
+  auto transformOp = cast<TransformOpInterface>(getOperation());
+
+  SmallVector<int64_t> tileSizes;
+  DiagnosedSilenceableFailure status = reifyMixedParamAndHandleResults(
+      state, transformOp, getMixedTileSizes(), tileSizes);
+  if (!status.succeeded())
+    return status;
+  SmallVector<int64_t> tileInterchange;
```

Comment:
This is the parameter that tells the transform how to do interchange, right? If so I don't understand how any of the `interchange` values can be handles to payload ops. When such a handle is to a payload op, you shouldn't be able to convert that op's result to a known-at-interpreter-time integer, right?

Reply:
Per `reifyMixedParamAndHandleResults`'s docs, I think understand now:
```
/// When possible, converts each `OpFoldResult` in `mixedResult` to
/// an integer if the value can be statically inferred.  If a result
/// is a `Value` then it must be either a `ParamType` or a handle
/// to an a constant like op.
```
That is, the only handles to payloads that are supported are to constant-like ops.